server/services/outOfServiceBedService.test.ts

```diff
@@ -72,17 +74,19 @@ describe('OutOfServiceBedService', () => {
   })
 
   describe('getAllOutOfServiceBeds', () => {
-    it('on success returns all outOfServiceBeds', async () => {
-      const expectedOutOfServiceBeds: Array<OutOfServiceBed> = outOfServiceBedFactory.buildList(2)
-
+    it('calls the get method on the outOfServiceBedClient with a page', async () => {
       const token = 'SOME_TOKEN'
-      outOfServiceBedClient.get.mockResolvedValue(expectedOutOfServiceBeds)
 
-      const outOfServiceBeds = await service.getAllOutOfServiceBeds(token)
+      const response = paginatedResponseFactory.build({
+        data: outOfServiceBedFactory.buildList(4),
```

Comment:
V minor but would prefer if we could build a list of here to reduce memory usage in tests

Reply:
as in, a smaller list?

Reply:
yeah `outOfServiceBedFactory.buildList(4)`. It feel pedantic writing it but when there's 2000+ unit tests it all adds up.

Reply:
yep ok